examples/landsat-c2-l1/LM01_L1GS_001010_19720908_02_T2/LM01_L1GS_001010_19720908_02_T2.json

```diff
@@ -0,0 +1,301 @@
+{
+  "type": "Feature",
+  "stac_version": "1.0.0",
+  "id": "LM01_L1GS_001010_19720908_02_T2",
+  "properties": {
+    "platform": "landsat-1",
+    "instruments": [
+      "mss"
+    ],
+    "created": "2022-03-07T13:41:20.119274Z",
+    "gsd": 60,
+    "description": "Landsat Collection 2 Level-1 Data Product",
+    "eo:cloud_cover": 43.0,
+    "view:off_nadir": 0,
+    "view:sun_elevation": 24.87312023,
+    "view:sun_azimuth": 172.41815593,
+    "proj:epsg": 32625,
+    "proj:shape": [
+      4214,
+      4296
+    ],
+    "proj:transform": [
+      60.0,
+      0.0,
+      358830.0,
+      0.0,
+      -60.0,
+      7953510.0
+    ],
+    "sci:doi": "10.5066/P9AF14YV",
+    "landsat:cloud_cover_land": 46.0,
+    "landsat:wrs_type": "1",
+    "landsat:wrs_path": "001",
+    "landsat:wrs_row": "010",
+    "landsat:collection_category": "T2",
+    "landsat:collection_number": "02",
+    "landsat:scene_id": "LM10010101972252XXX01",
+    "landsat:correction": "L1GS",
+    "datetime": "1972-09-08T13:43:34.091000Z"
+  },
+  "geometry": {
+    "type": "Polygon",
+    "coordinates": [
+      [
+        [
+          -34.342777359690906,
+          71.6736886240419
+        ],
+        [
+          -36.70080544861747,
+          70.18448226408043
+        ],
+        [
+          -32.42683403196701,
+          69.42483853001222
+        ],
+        [
+          -29.841815210171347,
+          70.85738280018273
+        ],
+        [
+          -34.342777359690906,
+          71.6736886240419
+        ]
+      ]
+    ]
+  },
+  "links": [
+    {
+      "rel": "cite-as",
+      "href": "https://doi.org/10.5066/P9AF14YV"
+    },
+    {
+      "rel": "alternate",
+      "href": "https://landsatlook.usgs.gov/stac-browser/collection02/level-1/standard/mss/1972/001/010/LM01_L1GS_001010_19720908_20200909_02_T2",
+      "type": "text/html",
+      "title": "USGS stac-browser page"
+    },
+    {
+      "rel": "root",
+      "href": "../collection.json",
+      "type": "application/json",
+      "title": "Landsat Collection 2 Level-1"
+    },
+    {
+      "rel": "collection",
+      "href": "../collection.json",
+      "type": "application/json",
+      "title": "Landsat Collection 2 Level-1"
+    },
+    {
+      "rel": "parent",
+      "href": "../collection.json",
+      "type": "application/json",
+      "title": "Landsat Collection 2 Level-1"
+    }
+  ],
+  "assets": {
+    "thumbnail": {
+      "href": "tests/data-files/mss/LM01_L1GS_001010_19720908_20200909_02_T2_thumb_small.jpeg",
```

Comment:
Still need to make these asset hrefs relative to the file (i.e. make them absolute then use `make_asset_hrefs_relative`).

Reply:
OK, this is sinking in. I am now making the asset hrefs absolute during asset creation time, which allows the `make_all_asset_hrefs_relative()` call to do it's thing when creating collections.